exporter/sentryexporter/README.md

```diff
@@ -13,6 +13,7 @@ For more details about distributed tracing in Sentry, please view [our documenta
 The following configuration options are supported:
 
 - `dsn`: The DSN tells the exporter where to send the events. You can find a Sentry project DSN in the “Client Keys” section of the “Project Settings” section of a Sentry project.
+- `env`: When the value is set, it will set the event environment tag, so the event can be filtered accordingly in Sentry.
```

Comment:
```suggestion
- `env`: When the value is set, it will set the event environment tag, so the event can be filtered accordingly in Sentry. Note that this applies to every single event that is processed by the Sentry Exporter.
```